### PR TITLE
fix(docs): fix snippets in Layout docs

### PR DESCRIPTION
**Pull request by layershifter**
image

Fixes issue in https://stardust-ui.github.io/react/layout.
The root reason is missing display names in production.

### CODE REVIEW

File: docs/src/components/ExampleSnippet/ExampleSnippet.tsx [hunk 1]
```diff
@@ -24,6 +24,13 @@ const ExampleSnippet = ({ render = () => null, value }: ExampleSnippetProps) =>
   const string =
     value ||
     reactElementToJSXString(element, {
+      displayName: (element: React.ReactElement<any>): string =>
```
Comment (layershifter) [inline docs/src/components/ExampleSnippet/ExampleSnippet.tsx:27]: https://github.com/algolia/react-element-to-jsx-string#reactelementtojsxstringreactelement-options

Comment (layershifter) [reply docs/src/components/ExampleSnippet/ExampleSnippet.tsx:27]: Original function uses `No Display Name`, that breaks `prettier.parser.babylon`

https://github.com/algolia/react-element-to-jsx-string/blob/0b17eaf3decafac13f276e705d4c850418cd4bb7/src/parser/parseReactElement.js#L19

**Review by miroslavstastny — APPROVED** [review]
Changelog entry for DOC?

**Comment by layershifter** [comment]
> Changelog entry for DOC?

Not sure, this looks more like a hotfix...

File: docs/src/views/Layout.tsx [hunk 2]
```diff
@@ -76,6 +76,7 @@ export default () => (
           </OpaqueBackground>
         )
       }}
+      value={`<OpaqueBackground color="lightyellow">This is a bad approach to opaque background :(</OpaqueBackground>`}
```
Comment (kuzhelov) [inline docs/src/views/Layout.tsx:79]: for this component we need to introduce display name as well, will address in a separate PR

**Pull request merged**